sys/include/net/gnrc/rpl.h

```diff
@@ -279,22 +284,25 @@ static inline bool GNRC_RPL_COUNTER_GREATER_THAN(uint8_t A, uint8_t B)
  * default lifetime will be multiplied by the lifetime unit to obtain the resulting lifetime
  * @{
  */
-#define GNRC_RPL_DEFAULT_LIFETIME (60)
-#define GNRC_RPL_LIFETIME_UNIT (2)
+#ifndef GNRC_RPL_DEFAULT_LIFETIME
+#define GNRC_RPL_DEFAULT_LIFETIME (5)
```

Comment:
The change of the value should be in a seperate PR than "allow to modify params in makefile"